lib/document/templates.js

```diff
@@ -124,17 +124,33 @@ ${faDom.css()}
 }
 
 const syntaxHighlighterHead = (node, syntaxHighlighter, attrs) => {
+  let header = ''
   if (syntaxHighlighter !== Opal.nil && syntaxHighlighter['$docinfo?']('head')) {
-    return syntaxHighlighter['$docinfo']('head', node, Opal.hash(attrs))
+    header = header + syntaxHighlighter['$docinfo']('head', node, Opal.hash(attrs))
```

Comment:
at this point `header` is empty so:

```suggestion
    header = syntaxHighlighter['$docinfo']('head', node, Opal.hash(attrs))
```

Reply:
I tried to make it symmetric with the branch below.